src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/WinHttpHandler/WinHttpHandlerIntegration.cs

```diff
@@ -38,11 +39,15 @@ public class WinHttpHandlerIntegration
         public static CallTargetState OnMethodBegin<TTarget, TRequest>(TTarget instance, TRequest requestMessage, CancellationToken cancellationToken)
             where TRequest : IHttpRequestMessage
         {
+            Log.Warning("WinHttpHandler started");
```

Comment:
Seems like these logs are for debugging.

Reply:
oops! you caught me

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1170/commits/e5d6c21afb0e573e74d328dc65ad6c51dd34094b